streamparse/ext/util.py

```diff
@@ -96,3 +96,9 @@ def get_nimbus_for_env_config(env_config):
         port = 6627
 
     return (host, port)
+
+
+def check_if_use_ssh_for_nimbus(env_config):
```

Comment:
I was thinking about good name for it, your variant is the best so far. :) Fixed.